src/views/createQuestionNumberView.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+import { CURRENT_QUESTION_NUM_ID } from '../constants.js';
+import { quizData } from '../data.js';
+
+/**
+ * Create a full question element
+ * @returns {Element}
+ */
+export const progressBar = () => {
+  const element = document.createElement('div');
+  // const storedCurrentQuestion = localStorage.getItem("currentIndex");
+  // if(!storedCurrentQuestion) return;
+  //quizData.currentQuestionIndex=+storedCurrentQuestion
+
+  element.innerHTML = String.raw`
+  
+  <p> Question <span id='${CURRENT_QUESTION_NUM_ID}'> ${
+    quizData.currentQuestionIndex + 1
+  } </span> of <span> ${quizData.questions.length} </span></p>
+   <div class='progressBarFull ></div>
```

Comment:
The class name is missing the closing quote.
```suggestion
   <div class='progressBarFull'></div>
```